ux.symfony.com/src/Command/LoadDataCommand.php

```diff
@@ -38,13 +39,20 @@ protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
 
-        $this->clearEntity(Chat::class, $io);
-        $this->clearEntity(TodoItem::class, $io);
-        $this->clearEntity(TodoList::class, $io);
-        $this->clearEntity(InvoiceItem::class, $io);
-        $this->clearEntity(Invoice::class, $io);
-        $this->growFood($io);
-        $this->manufactureProducts($io);
+        $progressIndicator = new ProgressIndicator($output, finishedIndicatorValue: '✅');
+        try {
+            $progressIndicator->start("Loading data");
+            $this->clearEntity(Chat::class, $io);
+            $this->clearEntity(TodoItem::class, $io);
+            $this->clearEntity(TodoList::class, $io);
+            $this->clearEntity(InvoiceItem::class, $io);
+            $this->clearEntity(Invoice::class, $io);
+            $this->growFood($io);
+            $this->manufactureProducts($io);
+            $progressIndicator->finish('Finished');
+        } catch (\Exception) {
+            $progressIndicator->finish('Failed', '🚨');
+        }
```

Comment:
I don't think we wan't to try/catch the code, as it will hide the exception if any of them is thrown:
```suggestion
        $progressIndicator->start("Loading data");
        $this->clearEntity(Chat::class, $io);
        $this->clearEntity(TodoItem::class, $io);
        $this->clearEntity(TodoList::class, $io);
        $this->clearEntity(InvoiceItem::class, $io);
        $this->clearEntity(Invoice::class, $io);
        $this->growFood($io);
        $this->manufactureProducts($io);
        $progressIndicator->finish('Finished');
```